dss-api.yml

```diff
@@ -51,8 +51,37 @@ info:
 
     ```
 
-    # Special String Formats
+    # Subscrptions
+
+    DSS supports webhook subscriptions for data events like bundle creation and deletion. Webhooks are callbacks to a public
+    HTTPS endpoint provided by your application. When an event matching your subscription occurs, DSS will send a push
+    notification (via an HTTPS POST or PUT request), giving your application an up-to-date stream of system activity.
+
+    Subscriptions are delivered with the payload format
+
+    ```
 
+    {
+      'transaction_id': {uuid},
+      'subscription_id': {uuid},
+      'event_type': "CREATE"|"TOMBSTONE"|"DELETE",
```

Comment:
This is a good point, as far as I can tell, the current analysis listener doesn't distinguish these types 🤔 Maybe this is why we had some duplicated workflows recently? @samanehsan